openshift-knative-operator/pkg/monitoring/rbac_proxy_injection.go

```diff
@@ -32,6 +32,13 @@ func InjectRbacProxyContainerToDeployments(deployments sets.String) mf.Transform
 			firstContainer := &dep.Spec.Template.Spec.Containers[0]
 			// Make sure we export metrics only locally
 			firstContainer.Env = append(firstContainer.Env, corev1.EnvVar{Name: "METRICS_PROMETHEUS_HOST", Value: "127.0.0.1"})
+			if u.GetName() == "kafka-ch-controller" {
+				firstContainer.Env = append(firstContainer.Env, corev1.EnvVar{Name: rbacProxyImageEnvVar, Value: getRbacProxyImage()})
```

Comment:
We need to propagate some values since now the dispatcher is not managed by the knative operator.